__init__.py

```diff
@@ -0,0 +1,5 @@
+from .sammo import Sammo
```

Comment:
Can we add a header to Python files please? Something like:

``` python
# coding: utf8

__contact__ = "info@hytech-imaging.fr"
__copyright__ = "Copyright (c) 2021 Hytech Imaging"
```

Reply:
Done